.rubocop.yml

```diff
@@ -13,41 +13,118 @@ Bundler/OrderedGems:
   Enabled: false
 Gemspec/OrderedDependencies:
   Enabled: false
+Layout/ClosingParenthesisIndentation:
+  Enabled: false
+Layout/FirstArrayElementIndentation:
+  EnforcedStyle: consistent
 Layout/LineLength:
+  Max: 100
+  Exclude:
+    - "spec/**/*_spec.rb"
+Layout/SpaceInLambdaLiteral:
   Enabled: false
-Lint/StructNewOverride:
+Layout/MultilineMethodCallIndentation:
+  Enabled: true
+  EnforcedStyle: indented
+Layout/SpaceInsideHashLiteralBraces:
   Enabled: true
+  EnforcedStyle: no_space
+  EnforcedStyleForEmptyBraces: no_space
+Lint/BooleanSymbol:
+  Enabled: false
+Lint/StructNewOverride:
+  Enabled: false
 Lint/RaiseException:
+  Enabled: false
+Lint/SuppressedException:
+  Exclude:
+    - "spec/spec_helper.rb"
+Metrics/AbcSize:
+  Max: 25
+Metrics/CyclomaticComplexity:
   Enabled: true
+  Max: 12
+Metrics/MethodLength:
+  Enabled: false
```

Comment:
@timriley Isn't this a good metric to remind the writer about concise method implementations?

Reply:
We have 16 instances of `# rubocop:disable Metrics/MethodLength` in hanami/hanami alone. This tells me that we're pretty much just ignoring this rule anyway. And the fact that we're reviewing each others code and acknowledging that the disabling of the rule is justified shows that it's just not helping us.

By disabling this rule we reduce a whole lot of line noise from the code, as well as friction when working within this repo.

I think our approach here should be continue to exercise thorough code review and regular refactoring towards the cleanest code possible. This is something only we as maintainers can do, and a robot telling us about it isn't really going to help.